grabbit/src/main/groovy/com/twcable/grabbit/client/batch/ClientBatchJob.groovy

```diff
@@ -42,8 +42,9 @@ class ClientBatchJob {
     public static final String WORKFLOW_CONFIGS = "workflowConfigIds"
     public static final String HOST = "host"
     public static final String PORT = "port"
-    public static final String USERNAME = "username"
-    public static final String PASSWORD = "password"
+    public static final String SERVER_USERNAME = "serverUsername"
+    public static final String SERVER_PASSWORD = "serverPassword"
+    public static final String CLIENT_USERNAME = "clientUsername"
```

Comment:
Because a `Grabbit Server` user only needs to `read` and not write.. and `Client` user needs to write, I think that would not be a sound assumption.